src/main/resources/view/PawbookTheme.css

```diff
@@ -308,7 +320,7 @@
     -fx-padding: 8 1 8 1;
 }
 
-#cardPane {
+#cardPaneDog {
```

Comment:
is this a mistake? I don't think I see any cardPaneDog around

Reply:
Fixed.